lambdas/service/app.py

```diff
@@ -244,6 +248,10 @@
             'name': 'Manifests',
             'description': 'Complete listing of files matching a given filter in TSV and other formats'
         },
+        {
+            'name': 'TDR',
```

Comment:
```suggestion
            'name': 'Repository',
```

We're trying to encapsulate repository specifics in the plugin.